package.json

```diff
@@ -1,123 +1,127 @@
-        "document-model": "^1.0.30",
-        "eslint": "^8.56.0",
-        "eslint-config-prettier": "^9.1.0",
-        "eslint-plugin-import": "^2.29.1",
-        "eslint-plugin-jest-dom": "^5.1.0",
-        "eslint-plugin-n": "^16.6.2",
-        "eslint-plugin-prettier": "^5.1.3",
-        "eslint-plugin-promise": "^6.1.1",
-        "eslint-plugin-react": "^7.33.2",
-        "eslint-plugin-react-hooks": "^4.6.0",
-        "eslint-plugin-storybook": "^0.6.15",
-        "eslint-plugin-tailwindcss": "^3.14.2",
-        "fast-xml-parser": "^4.3.4",
-        "husky": "^9.0.11",
-        "jsdom": "^24.0.0",
-        "lint-staged": "^15.2.2",
-        "natural-orderby": "^3.0.2",
-        "postcss": "^8.4.35",
-        "prettier": "^3.2.5",
-        "prettier-plugin-organize-imports": "^3.2.4",
-        "react": "^18.2.0",
-        "react-aria-components": "1.1.0",
-        "react-aria": "^3.32.0",
-        "react-click-away-listener": "^2.2.3",
-        "react-dom": "^18.2.0",
-        "semantic-release": "^23.0.2",
-        "storybook": "^7.6.15",
-        "storybook-addon-pseudo-states": "^2.1.2",
-        "tailwind-merge": "^2.2.1",
-        "tailwindcss": "^3.4.1",
-        "tailwindcss-animate": "^1.0.7",
-        "tailwindcss-react-aria-components": "1.1.0",
-        "tsx": "^4.7.1",
-        "typescript": "^5.3.3",
-        "usehooks-ts": "^2.14.0",
-        "vite": "^5.1.2",
-        "vite-plugin-dts": "^3.7.2",
-        "vitest": "^1.2.2"
-    },
-    "peerDependencies": {
-        "react": "^18.2.0",
-        "react-dom": "^18.2.0",
-        "react-aria-components": "1.1.0",
-        "react-aria": "^3.32.0"
-    },
-    "resolutions": {
-        "jackspeak": "2.1.1"
-    },
-    "dependencies": {
-        "@internationalized/date": "^3.5.1",
-        "@radix-ui/react-dialog": "^1.0.5",
-        "@react-input/number-format": "^1.0.21",
-        "change-case": "^5.4.3",
-        "currency.js": "^2.0.4",
-        "date-fns": "^3.3.1",
-        "react-hook-form": "^7.50.1"
-    }
+  "name": "@powerhousedao/design-system",
+  "version": "1.0.0-alpha.61",
+  "description": "",
+  "files": [
+    "/dist"
+  ],
+  "type": "module",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "create-icon-name-type": "tsx scripts/create-icon-type-names-definition.ts",
+    "prepublish": "npm run create-icon-name-type",
+    "build": "npm run create-icon-name-type && vite build",
+    "storybook": "npm run create-icon-name-type && storybook dev -p 6006",
+    "build-storybook": "npm run create-icon-name-type && storybook build",
+    "check-types": "tsc",
+    "lint": "eslint . && npm run check-types",
+    "lint:fix": "eslint --fix . && npm run check-types",
+    "test": "vitest run",
+    "test:watch": "vitest watch",
+    "release": "semantic-release",
+    "prepare": "husky install",
+    "chromatic": "npx chromatic --project-token=chpt_b9c79db0df52861"
```

Comment:
Should the project token be a repo secret?

Reply:
haha yeah that also freaked me out the first time I saw it. This is actually added to the package.json automatically when you publish to chromatic for the first time. This token is just to identify the project on chromatic, it doesn't have any kind of access privileges. So having it here is all good, if not a bit weird lol